src/main/scala/it/pagopa/interop/purposemanagement/api/impl/ResponseHandlers.scala

```diff
@@ -119,6 +119,7 @@ object ResponseHandlers extends AkkaResponses {
     result match {
       case Success(s)                                    => success(s)
       case Failure(ex: PurposeNotFound)                  => notFound(ex, logMessage)
+      case Failure(ex: DraftPurposeVersionNotFound)      => notFound(ex, logMessage)
```

Comment:
we do not expect to receive this error because the draft should already exist.
Let's remove this line so that a 500 is raised and we can be notified by the alarm
```suggestion
```